python/pyspark/ml/torch/distributor.py

```diff
@@ -263,9 +263,23 @@ class TorchDistributor(Distributor):
 
     .. versionadded:: 3.4.0
 
+    Parameters
```

Comment:
This has been duplicated (since it's also defined in the `__init__` function, but that isn't actually shown in the documentation). Should I remove it from the `__init__` section area?